tests/testthat/test-underscored.R

```diff
@@ -2,7 +2,7 @@ context("Deprecated SE variants")
 
 test_that("complete_()", {
   df <- tibble(x = 1:2, y = 1:2, z = 3:4)
-  out <- complete_(df, list("x", ~y))
+  out <- complete_(df, list("x", ~ y))
```

Comment:
Can you please remove the space? See http://style.tidyverse.org/syntax.html#spacing

Reply:
Done, in 2 places